manager/controllers/advisory_detail_test.go

```diff
@@ -122,3 +123,18 @@ func TestAdvisoryDetailCachePreloading(t *testing.T) {
 	assert.True(t, ok)
 	advisoryDetailCache.Purge()
 }
+
+// nolint: errcheck
```

Comment:
I think you can use:
~~~golang
var errResp utils.ErrorResponse
ParseReponseBody(t, w.Body.Bytes(), &errResp)
assert.Equal(t, InvalidOffsetMsg, errResp.Error)
~~~
... as used elsewhere so you don't need to use this "nolint" annotation.